lib/formatters/JsonFormatter.js

```diff
@@ -60,11 +61,10 @@ class JsonFormatter extends BaseFormatter {
         id: obj.requestId
       }
     };
-
+    const meta = merge(metaBody, metaHeaders);
```

Comment:
❓ Why are you merging body with headers?

Reply:
I spent 30' yesterday to find ouy what was going on here. 
metaBody has value `{request: {body: {...}}}` and metaHeaders has value `{request: {headers: {...}}}`, so after extended the two objects, headers were overriding the body object.

So I had to merge